modules/virt-deleting-virt-cli.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * virt/install/uninstalling-virt.adoc
+
+[id="virt-deleting-virt-cli_{context}"]
+= Deleting the {VirtProductName} using the CLI
+
+You can delete {VirtProductName} by using the CLI.
+
+.Prerequisites
+
+- `oc` command installed on workstation.
+
+- Access to an {product-title} cluster using an account with
+`cluster-admin` permissions.
+
+.Procedure
+
+. Delete the HyperConverged Custom Resource:
++
+----
+$ oc delete HyperConverged kubevirt-hyperconverged -n openshift-cnv
+----
+
+. Delete the subscription:
++
+----
+$ oc delete subscription kubevirt-hyperconverged -n openshift-cnv
+----
+
+. Extract the ClusterServiceVersion (CSV) name for  {VirtProductName}:
```

Comment:
You have an extra space here. You're also setting it as a variable, which I think it more important. I'd say something like:
> Set the ClusterServiceVersion (CSV) name for {VirtProductName} as an environment variable:

Reply:
Updated the content.